build-system/pr-check/checks.js

```diff
@@ -78,6 +79,7 @@ async function prBuildWorkflow() {
   // Check document links only for PR builds.
```

Comment:
This comment only applies to `check-links`, so move it to line 81. (Inline and shortened is fine.)